### ISSUE TITLE
Korean IME support broken (when other languages installed)

### ISSUE THREAD

**Issue by bermlee** [Microsoft/vscode#62642]
Korean support (with other languages and layouts installed, like Dvorak) is atrocious in VS Code  running on Windows.

IME does not engage, or when it ever does, typing the four left home row keys result in 매뎌 instead of ㅁㄴㅇㄹ (for the keys ASDF), because on Dvorak its AOEU. 

This same bug happens in Atom, and in Chrome. For Atom and Chrome, Alt-Tabing away to another app and coming back, then switching IMEs again to Korean might make it type correctly. Edge, Firefox do not have this problem. I think it may be a problem with Electron, or Chromium?

**Comment by alexandrudima** [comment]
If you can reproduce this in Chrome, it is a Chromium issue. Please create an issue against https://bugs.chromium.org/p/chromium/issues/list

**Comment by bermlee** [comment]
Reported at: https://bugs.chromium.org/p/chromium/issues/detail?id=905222&can=2&q=component%3ABlink%3EEditing%3EIME%20&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified

**Comment by bermlee** [comment]
Just FYI: I assume this problem arose because of two things: the Microsoft Korean IME is still a hack-- instead of taking in pure keyboard codes, it's retranslating QWERTY input into the corresponding Korean, and later some smartypants on the Chromium side wanted to use Dvorak to input Chinese Pinyin (not a bad idea), and told it to ignore the Windows IME. I finally got around to reporting this because more and more apps are using the Electron framework, and I really liked using VS Code.

**Comment by bermlee** [comment]
As I was trying to reproduce the issue, I discovered the buggy behavior is gone! Did someone fix it, either at Windows, Electron, Atom or even VS Code? 

This is amazing! Everything works the way it did nearly a year ago. I had resorted to stop using Chrome or Atom whenever I needed it, but using more apps that were built with Electron (the last straw was Twistapp) I decided to try to bring it to someone's attention. 

Thank you. I hope it's not too early to rejoice. I'll keep testing for a few more days, on other machines as well. 

FYI, I had opened a ticket at: 
https://bugs.chromium.org/p/chromium/issues/detail?id=905222&can=2&q=component%3ABlink%3EEditing%3EIME%20&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified

Then I also opened a ticket at Windows Feedback hub (since it's also an Korean IME issue, might even be solved by modernizing the Korean IME to not translate from expected QWERTY input, and take in pure Keyboard codes instead) https://aka.ms/AA373vb
